doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+
+Once an IO device is directly assigned to a workload, that workload will then have an exclusive access to that device and no other entities (including the hypervisor) can access it.
+
+This method provides better performance than the para-virtualized one as no hypervisor is involved but provides less flexibility and less portability.
+
+Having an IO device directly assigned to a workload means that the workload needs to run vendor specific drivers and libraries to be able to access that device which makes the workload less portable and dependent on a specific hardware type from a specific vendor which is not aligned with the overall strategy and goals of CNTT and hence this method of IO Virtualization should not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+
+<p align="center"><img src="./figures/tech_vtd.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 5:</b> Direct Assignment with Virtual Technology.</p>
+
+<a name="4.1.3"></a>
+#### 4.1.3 Device Sharing with SR-IOV & Virtual Technology.
+
+Unlike Direct assignment, this method allows a hardware device to be shared amongst many workloads where each workload has an exclusive access to one region of the device (known as VF). 
+
+For this method to be possible, the IO device need to support Single Root Input Output Virtualization (SR-IOV) which allows it to present itself as multiple devices, known as Physical Functions, PFs, and Virtual Functions, VFs as presented in **Figure 6**.
+
+Each of those Virtual Functions can then be independently assigned exclusively to a workload (with the appropriate support by VT-d or AMD-V from the infrastructure).
+
+Similar to the previous method ("Direct Assignment"), this method provides better performance than the para-virtualized one but lacks the flexibility and the portability sought and therefore should also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+
+<p align="center"><img src="./figures/tech_sriov.png" alt="sriov" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 6:</b> Device Sharing with SR-IOV & Direct Assignment.</p>
+
+<a name="4.1.4"></a>
+#### 4.1.4 Para-Virtualization method (Hardware support)
+
+This method basically is a mixture between the software only para-virtualization method and the direct assignment method (including the device sharing method) where the frontEnd driver which is running on the workload is a standard off the shelf driver whereas the backEnd driver is implemented straight in hardware (by-passing the hypervisor with support from VT-d/AMD-V and SR-IOV in case of device sharing) as shown in **Figure 7**.
```

Comment:
Consider replacing proprietary terms w/ "...bypassing the hypervisor with hardware support from an IOMMU and SR-IOV..."
"whereas" -> consider replacing w/ "and"
"the backEnd driver is implemented straight in hardware" // This is hard to understand; can we clarify?

Reply:
fixed